generators/server/templates/src/main/resources/config/liquibase/master.xml.ejs

```diff
@@ -12,11 +12,15 @@
     <property name="clobType" value="varchar(1000000)" dbms="h2"/>
     <property name="blobType" value="varchar(1000000)" dbms="h2"/>
   <%_ } else { _%>
+    <%_ if (prodDatabaseTypePostgres) { _%>
     <property name="clobType" value="longvarchar" dbms="h2"/>
```

Comment:
Hmmm, strange thing: clobtype being of a kind for h2 when databasetype is postgres?

Reply:
Matches postgresql prod :
https://github.com/jhipster/generator-jhipster/blob/4c6a1884870ba04b2f4b889db323aa8974b2cf80/generators/server/templates/src/main/resources/config/liquibase/master.xml.ejs#L42

and old behavior:
https://github.com/jhipster/generator-jhipster/blob/63dfdf81d1cc8c522be01014c10b80c5fab72708/generators/server/templates/src/main/resources/config/liquibase/master.xml.ejs#L22-L24